app/products/agents/components/agent_post/index.tsx

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {regenerateResponse, stopGeneration} from '@agents/actions/remote/generation_controls';
+import streamingStore from '@agents/store/streaming_store';
+import {StreamingEvents, type Annotation, type StreamingState, type ToolCall} from '@agents/types';
+import {isPostRequester} from '@agents/utils';
+import React, {useCallback, useEffect, useMemo, useState} from 'react';
+import {DeviceEventEmitter, StyleSheet, View} from 'react-native';
+
+import FormattedText from '@components/formatted_text';
+import Markdown from '@components/markdown';
+import {SNACK_BAR_TYPE} from '@constants/snack_bar';
+import {useServerUrl} from '@context/server';
+import {useTheme} from '@context/theme';
+import {safeParseJSON} from '@utils/helpers';
+import {showSnackBar} from '@utils/snack_bar';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+import CitationsList from '../citations_list';
+import ControlsBar from '../controls_bar';
+import ReasoningDisplay from '../reasoning_display';
+import ToolApprovalSet from '../tool_approval_set';
+
+import StreamingIndicator from './streaming_indicator';
+
+import type PostModel from '@typings/database/models/servers/post';
+import type {AvailableScreens} from '@typings/screens/navigation';
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+    return StyleSheet.create({
+        container: {
+            flex: 1,
+        },
+        messageContainer: {
+            flexDirection: 'row',
+            flexWrap: 'wrap',
+            alignItems: 'flex-start',
+        },
+        messageText: {
+            color: theme.centerChannelColor,
+            fontSize: 15,
+            lineHeight: 20,
+        },
+        precontentContainer: {
+            flexDirection: 'row',
+            alignItems: 'center',
+            paddingVertical: 8,
+        },
+        precontentText: {
+            color: changeOpacity(theme.centerChannelColor, 0.6),
+            fontSize: 14,
+            fontStyle: 'italic',
+            marginRight: 8,
+        },
+    });
+});
```

Comment:
```suggestion
const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
    return {
        container: {
            flex: 1,
        },
        messageContainer: {
            flexDirection: 'row',
            flexWrap: 'wrap',
            alignItems: 'flex-start',
        },
        messageText: {
            color: theme.centerChannelColor,
            fontSize: 15,
            lineHeight: 20,
        },
        precontentContainer: {
            flexDirection: 'row',
            alignItems: 'center',
            paddingVertical: 8,
        },
        precontentText: {
            color: changeOpacity(theme.centerChannelColor, 0.6),
            fontSize: 14,
            fontStyle: 'italic',
            marginRight: 8,
        },
    };
});
```

not a blocker, but we don't really need StyleSheet.create